sharing/standard.go

```diff
@@ -56,6 +61,50 @@ func (handler *Handler) handleStandard(ctx context.Context, w http.ResponseWrite
 	}
 
 	pr.access = access
+
+	q := r.URL.Query()
+
+	// flag used for get locations request.
+	locationsFlag := queryFlagLookup(q, "locations", false)
+	if locationsFlag {
```

Comment:
Create a separate func for this e.g.

```
if locationsFlag {
    handler.serveLocations(ctx, uplink.Config{}, ....)
    return nil
}
```

But otherwise looks good.

Reply:
done